src/cli/README_JOINER.md

```diff
@@ -28,6 +29,25 @@ stop
 Done
 ```
 
+### discerner
+
+Usage: `joiner discerner [discerner]`
+
+Print or set the Joiner Discerner.
```

Comment:
Provide some note here that the discerner takes the place of the existing EUI-64 method?

Reply:
Yes, some high-level context would be great given the term `Joiner Discerner` isn't in the spec.  Please provide a definition of the term and possibly a use case example.  How will the joiner discerner be communicated from the Joiner to the Commissioner?